### DISCUSSION TITLE
try to install the latest version but got 1.8.4 instead on many windows machines

### DISCUSSION THREAD

**Discussion by whitegreen** [shapely/shapely#1772] [Q&A]
Today we tried to install the latest version on ten windows machines, but always get the 1.8.4 version.
We used commands in pyCharm:
conda install shapely
and also tried:
pip3 install shapely, or pip install shapely, or pip install shapely==2.0.1
but always got the 1.8.4 version.
The pycharm package tool can find the shapely 2.0.1, but after pressing the install button, it dumps to the 1.8.4 version again. 
Any idea what's going on?

**Comment by jorisvandenbossche** [comment]
You might be using Python 3.6? 
Shapely 1.8.4 was the last version to provide binary wheel packages for that version of Python.
